cache/redis/redis_test.go

```diff
@@ -8,11 +8,11 @@ import (
 	"syscall"
 	"testing"
 
-	goredis "github.com/go-redis/redis"
 	"github.com/go-spatial/tegola/cache"
 	"github.com/go-spatial/tegola/cache/redis"
 	"github.com/go-spatial/tegola/dict"
 	"github.com/go-spatial/tegola/internal/ttools"
+	goredis "github.com/redis/go-redis/v9"
```

Comment:
nit, but let's move `goredis "github.com/redis/go-redis/v9"` aboce the `go-spatial` imports but below the standard lib imports. e.g.:

```go
import (
	"crypto/tls"
	"net"
	"os"
	"reflect"
	"syscall"
	"testing"

         goredis "github.com/redis/go-redis/v9"

	"github.com/go-spatial/tegola/cache"
	"github.com/go-spatial/tegola/cache/redis"
	"github.com/go-spatial/tegola/dict"
	"github.com/go-spatial/tegola/internal/ttools"
)
```